deploy/single-namespace/deployment.yaml

```diff
@@ -24,4 +24,9 @@ spec:
         args:
         - --exposer-image=quay.io/tnozicka/openshift-acme:exposer
         - --loglevel=4
-        - --namespace=acme-controller
+        - --namespace=$(CURRENT_NAMESPACE)
```

Comment:
Thanks for sending the PR. Wanna get the other one as well?
https://github.com/tnozicka/openshift-acme/blob/master/deploy/specific-namespaces/deployment.yaml#L27

Reply:
Yes, why not. Updated.